_includes/components/search-results/SearchResult/partials/getLandingResults.js

```diff
@@ -1,5 +1,5 @@
 import fetchWrapper from './../../../../utilities/js/helpers/fetchWrapper.js';
-import normaliseReturn from './normaliseReturn.js';
+import normaliseServerResponse from './normaliseServerResponse.js';
```

Comment:
Cool - this makes more sense - now that I understand what it does, I think normaliseSearchResponse would make more sense - server could be any server. Not critical, just a suggestion